src/frontend/game/components/PlayerList.tsx

```diff
@@ -1,10 +1,9 @@
 import { type FunctionalComponent } from 'preact';
 import { useClientContext } from 'src/frontend/game/ClientContextProvider.tsx';
-import { IncreaseMyScore } from 'src/frontend/game/components/IncreaseMyScore';
```

Comment:
if you're using this pr to learn how to create a new game event this file can be ignored, the changes are unrelated to implementing the ChangePlayerNameEvent.